sql/moz-fx-data-shared-prod/subscription_platform_backend_cirrus_derived/delete_events_v1/query.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TEMP FUNCTION format_uuid(value STRING)
+RETURNS STRING AS (
+  CONCAT(
+    SUBSTR(value, 1, 8),
+    "-",
+    SUBSTR(value, 9, 4),
+    "-",
+    SUBSTR(value, 13, 4),
+    "-",
+    SUBSTR(value, 17, 4),
+    "-",
+    SUBSTR(value, 21, 12)
+  )
+);
+
+CREATE TEMP FUNCTION uuid_v5(value STRING, namespace STRING)
+RETURNS STRING AS (
+  FORMAT_UUID(
+    TO_HEX(
+      LEFT(
+        SHA1(CONCAT(FROM_HEX(REPLACE(namespace, "-", "")), CAST(value AS BYTES FORMAT 'UTF8'))),
+        16
+      ) --
+      & FROM_HEX('ffffffffffff0fff3fffffffffffffff') --
+      | FROM_HEX('00000000000050008000000000000000')
+    )
+  )
+);
+
+WITH subplat_namespaces AS (
+  SELECT
+    CAST(
+      AEAD.DECRYPT_BYTES(
+        (SELECT keyset FROM `moz-fx-dataops-secrets.airflow_query_keys.subplat_namespaces_prod`),
+        ciphertext,
+        CAST(key_id AS BYTES)
+      ) AS STRING
+    ) AS namespace
+  FROM
+    `moz-fx-data-shared-prod.subscription_platform_derived.encrypted_keys_v1`
```

Comment:
I think a basic `metadata.yaml` file ought to be added for this table (a la [sql/moz-fx-data-shared-prod/firefox_accounts_derived/encrypted_keys_v1/metadata.yaml](https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/firefox_accounts_derived/encrypted_keys_v1/metadata.yaml)), otherwise the BQ permissions automation is going to complain about not having that.